docs/tutorials/zarr/multiscale.md

```diff
@@ -84,11 +123,16 @@ group = zarr.open("out.zarr")
 With multiscale mode enabled, an image pyramid will be formed by rescaling the data by a factor of 2 progressively until the rescaled image is smaller than the specified zarr chunk size in both dimensions. In this example, the original image dimensions are (1920, 1080), and we chunked the data using tiles 1/3 of the size of the image, namely (640, 360). To illustrate this point, we'll inspect the sizes of the various levels in the multiscale data and compare it to our specified chunk size.
 
 ```python
-group["0"], group["1"], group["2"]
+print(group["0"])
 ```
 
 The output will be:
 
+```
+<zarr.core.Array '/0' (300, 3, 10, 1080, 1920) uint8>
+```
+
+TO BE UPDATED:
```

Comment:
Should this be here?

Reply:
This tutorial got added back when Melissa rebased. I will delete again now as discussed and we can add it back when the bug is fixed.